src/gatsby/src/components/park/contact.js

```diff
@@ -1,11 +1,117 @@
 import React from "react"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
 import HtmlContent from "./htmlContent"
 
-export default function Contact({ contact }) {
+// Helper function to get href prefix based on icon
```

Comment:
I'm not sure if it's intuitive that we are adding the scheme for tel/mailto and not http/https.  Maybe check with UX.